### DISCUSSION TITLE
Welcome to gbRelease Discussions!

### DISCUSSION THREAD

**Discussion by DanRunfola** [wmgeolab/geoBoundaries#740] [General]
## :earth_africa: :earth_americas: :earth_asia: :globe_with_meridians:  Welcome! :globe_with_meridians: :earth_africa: :earth_americas: :earth_asia: 
  We’re using Discussions as a place to connect with the users and contributors to geoBoundaries. While we will use issues on github (https://github.com/wmgeolab/gbRelease/issues) for most matters, for broad topics that don't fit into one issue (i.e., our product line, how we handle conflicted boundaries), we hope to use discussions so as to promote transparency. 

Please feel free to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded! geoBoundaries is only as strong as its community.

**Comment by leeberryman** [comment]
@DanRunfola what is the anticipated timeline before cgaz and other datasets are generated for the v4 release?

**Comment by DanRunfola** [comment]
I am very actively trying to get 4.0 out by the end of the month (August),
with a slight risk we slip into September.

Waiting on one more ADM2 to give us complete coverage, and then running the
final builds (after I figure out that Nepal issue you raised :) ).

**Reply by leeberryman** [reply]
Is there a punch list of ADM2's that are needed that I can help mobilize some more help on? @DanRunfola.

**Comment by DanRunfola** [comment]
Punch list is here - https://github.com/wmgeolab/geoBoundaries/milestone/3

It's just MDA ADM2, which I believe someone is already working on.  The
rest is just pieces I'm hammering on - the issue you raised about the ISO
code field in NPL ADM0 being incorrect is something I need to look at as
well, as that could impact the build code.

My goal is to have the final build products built over the weekend, and
then use those to build CGAZ4 next week.  That would give us a release of
4.0 mid / end of next week.  I *really *would like to get it out by Tuesday
(August 31), but it may slip a bit.

**Reply by leeberryman** [reply]
@DanRunfola great! Let me know if there are tangible ways we can help!  Would like to sit down in person sometime and hear your big picture vision for the project if you have time.  I am working on i18n data to correspond with the repo and would like to see if it could help you your users.

**Comment by DanRunfola** [comment]
Of course :).  I hope to hold something of a users forum at some point soon to start building community a bit more formally.  TBD on those details - coordinating with GRID3 colleagues up at Columbia (who are very helpful with coordination while we deal with the technical issues!).

Just FYSA, I (just now) clicked the final build button for 4.0, so on track to have it all together by next Tuesday.  Fingers crossed.

**Reply by leeberryman** [reply]
Great news! Good to hear about a more formal meeting in the mix.  The grid3 project is great too.  I happen to be down the road about an hour working on a project for a little while and could drop by W&M if you have time coffee & informal conversation.

**Comment by DanRunfola** [comment]
Definitely, though it'll have to wait until post-pandemic.  Things are still being held pretty close here until we get through the Fall.

**Reply by leeberryman** [reply]
👍
